src/AWSS3.jl

```diff
@@ -614,13 +619,16 @@ s3_purge_versions(a...) = s3_purge_versions(default_aws_config(), a...)
 # Optional Arguments
 - `data_type=`; `Content-Type` header.
 - `encoding=`; `Content-Encoding` header.
+- `access_permissions=`; 'x-amz-acl' header for setting access permissions with canned config.
```

Comment:
Would you be willing to rename this to just `acl` to more closely match the cli interface?

Reply:
Sure, if you want.  Something I'd like to ask you to consider though: for those who aren't constantly dealing with AWS the name `acl` is extremely mysterious.  I'm not sure I'd remember what that pertains to even justa few days from now.  Are you sure you want to call the keyword arg that?

Reply:
FWIW, ACLs are a pretty common concept outside of S3 as well. Most linux admins would also be familiar with them for managing default access permissions within a directory (which is a bit more intuitive than how S3 does it). The other alternative I can see is just `access_control_list`.

Reply:
Ok, good enough for me then, I'll take your word for it.

I'll change it when I get a chance.

Reply:
Thank you